### PR TITLE
Pioneer 14 deployment and recovery details

### PR DESCRIPTION
**Pull request by snwhite**
Pioneer 14 profiler mooring deployment and recovery details.

### CODE REVIEW

File: deployment/CP04OSPM_Deploy.csv [hunk 1]
```diff
@@ -91,21 +91,21 @@ AR34,,AR39,,CP04OSPM-WFP01-02-DOFSTK000,11,1,2019-04-22T14:13:00,2019-10-13T16:0
...
+AR44,,,,CP04OSPM-SBS01-00-STCENG000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,,CGCON-BSTC01-00004,39.943117,-70.88605,,0,438,
+AR44,,,,CP04OSPM-SBS11-02-MOPAK0000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,,CGINS-MOPAK0-11357,39.943117,-70.88605,,0,438,
+AR44,,,,CP04OSPM-WFP01-00-WFPENG000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGCON-CWFPCP-10303,39.943117,-70.88605,,417,438,
+AR44,,,,CP04OSPM-WFP01-01-VEL3DK000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGINS-VEL3DK-00036,39.943117,-70.88605,,417,438,
+AR44,,,,CP04OSPM-WFP01-02-DOFSTK000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGINS-DOFSTK-02725,39.943117,-70.88605,,417,438,
+AR44,,,,CP04OSPM-WFP01-03-CTDPFK000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGINS-CTDPFK-00126,39.943117,-70.88605,,417,438,
+AR44,,,,CP04OSPM-WFP01-04-FLORTK000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGINS-FLORTK-01118,39.943117,-70.88605,,417,438,
+AR44,,,,CP04OSPM-WFP01-05-PARADK000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGINS-PARADK-20468,39.943117,-70.88605,,417,438,
```
Comment (rtravis-whoi) [inline deployment/CP04OSPM_Deploy.csv:111]: looks good.  Are the sensor depths correct for previous deployment (OSPM-12)?

Comment (snwhite) [reply deployment/CP04OSPM_Deploy.csv:111]: For the deep site (nominally 460 m), the ADCP should be 45 m up from the seafloor.  So for a a 451 m deployment depth, the ADCP should be at 406, which it is.

File: deployment/CP05MOAS-PG583_Deploy.csv [hunk 2]
```diff
@@ -5,4 +5,11 @@ SI20170418,,TS20170727,,CP05MOAS-PG583-02-DOSTAM000,1,1,2017-04-19T05:30:47,2017
...
+TS20200613,,,,CP05MOAS-PG583-00-ENG000000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGCON-GLDRGP-00583,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-01-CTDGVM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-CTDGVM-09307,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-02-DOSTAM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-DOSTAM-00539,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-03-FLORTM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-FLORTM-04093,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-04-FLORTO000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-FLORTO-01348,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-05-NUTNRM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-NUTNRM-00709,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-06-PARADM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-PARADM-50218,40.08518,-70.36618,,200,200,CNPM
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-PG583_Deploy.csv:15]: looks good

Comment (snwhite) [reply deployment/CP05MOAS-PG583_Deploy.csv:15]: CGVEH-GLDRGP-00583  should be GLDRCP  (this needs to be added to the node_bulk_load-AssetRecord. 
CGCON-GLDRGP-00583 should be GLDRCP  (this already exists in the eng_bulk_load-AssetRecord.

File: deployment/CP02PMCI_Deploy.csv [hunk 3]
```diff
@@ -107,21 +107,21 @@ AR34,,AR39,,CP02PMCI-WFP01-02-DOFSTK000,12,1,2019-04-17T02:25:00,2019-10-07T20:0
...
+AR44,,,,CP02PMCI-SBS01-00-STCENG000,14,1,2020-06-12T13:40:00,,CGMCP-02PMCI-00014,,CGCON-BSTC01-00040,40.2267,-70.889033,,0,127,
+AR44,,,,CP02PMCI-SBS01-01-MOPAK0000,14,1,2020-06-12T13:40:00,,CGMCP-02PMCI-00014,,CGINS-MOPAK0-58484,40.2267,-70.889033,,0,127,
+AR44,,,,CP02PMCI-WFP01-00-WFPENG000,14,1,2020-06-12T13:40:00,,CGMCP-02PMCI-00014,CGVEH-CWFPCP-10302,CGCON-CWFPCP-10302,40.2267,-70.889033,,104,127,
+AR44,,,,CP02PMCI-WFP01-01-VEL3DK000,14,1,2020-06-12T13:40:00,,CGMCP-02PMCI-00014,CGVEH-CWFPCP-10302,CGINS-VEL3DK-00037,40.2267,-70.889033,,104,127,
+AR44,,,,CP02PMCI-WFP01-02-DOFSTK000,14,1,2020-06-12T13:40:00,,CGMCP-02PMCI-00014,CGVEH-CWFPCP-10302,CGINS-DOFSTK-02724,40.2267,-70.889033,,104,127,
+AR44,,,,CP02PMCI-WFP01-03-CTDPFK000,14,1,2020-06-12T13:40:00,,CGMCP-02PMCI-00014,CGVEH-CWFPCP-10302,CGINS-CTDPFK-00125,40.2267,-70.889033,,104,127,
+AR44,,,,CP02PMCI-WFP01-04-FLORTK000,14,1,2020-06-12T13:40:00,,CGMCP-02PMCI-00014,CGVEH-CWFPCP-10302,CGINS-FLORTK-01119,40.2267,-70.889033,,104,127,
+AR44,,,,CP02PMCI-WFP01-05-PARADK000,14,1,2020-06-12T13:40:00,,CGMCP-02PMCI-00014,CGVEH-CWFPCP-10302,CGINS-PARADK-20465,40.2267,-70.889033,,104,127,
```
Comment (cdobs) [inline deployment/CP02PMCI_Deploy.csv:127]: Looks good

File: deployment/CP02PMCO_Deploy.csv [hunk 4]
```diff
@@ -107,21 +107,21 @@ AR34,,AR39,,CP02PMCO-WFP01-02-DOFSTK000,12,1,2019-04-22T04:09:00,2019-10-08T12:0
...
+AR44,,,,CP02PMCO-SBS01-00-STCENG000,14,1,2020-06-10T13:53:00,,CGMCP-02PMCO-00014,,CGCON-BSTC01-00010,40.101083,-70.88765,,0,148,
+AR44,,,,CP02PMCO-SBS01-01-MOPAK0000,14,1,2020-06-10T13:53:00,,CGMCP-02PMCO-00014,,CGINS-MOPAK0-11366,40.101083,-70.88765,,0,148,
+AR44,,,,CP02PMCO-WFP01-00-WFPENG000,14,1,2020-06-10T13:53:00,,CGMCP-02PMCO-00014,CGVEH-CWFPCP-10305,CGCON-CWFPCP-10305,40.101083,-70.88765,,127,148,
+AR44,,,,CP02PMCO-WFP01-01-VEL3DK000,14,1,2020-06-10T13:53:00,,CGMCP-02PMCO-00014,CGVEH-CWFPCP-10305,CGINS-VEL3DK-00034,40.101083,-70.88765,,127,148,
+AR44,,,,CP02PMCO-WFP01-02-DOFSTK000,14,1,2020-06-10T13:53:00,,CGMCP-02PMCO-00014,CGVEH-CWFPCP-10305,CGINS-DOFSTK-02628,40.101083,-70.88765,,127,148,
+AR44,,,,CP02PMCO-WFP01-03-CTDPFK000,14,1,2020-06-10T13:53:00,,CGMCP-02PMCO-00014,CGVEH-CWFPCP-10305,CGINS-CTDPFK-00128,40.101083,-70.88765,,127,148,
+AR44,,,,CP02PMCO-WFP01-04-FLORTK000,14,1,2020-06-10T13:53:00,,CGMCP-02PMCO-00014,CGVEH-CWFPCP-10305,CGINS-FLORTK-01101,40.101083,-70.88765,,127,148,
+AR44,,,,CP02PMCO-WFP01-05-PARADK000,14,1,2020-06-10T13:53:00,,CGMCP-02PMCO-00014,CGVEH-CWFPCP-10305,CGINS-PARADK-20466,40.101083,-70.88765,,127,148,
```
[the remark this answers is not in the document]
Comment (cdobs) [reply deployment/CP02PMCO_Deploy.csv:127]: Looks good

File: deployment/CP02PMUI_Deploy.csv [hunk 5]
```diff
@@ -116,21 +116,21 @@ AR34,,AR39,,CP02PMUI-WFP01-02-DOFSTK000,13,1,2019-04-15T01:03:00,2019-10-06T19:1
...
+AR44,,,,CP02PMUI-SBS01-00-STCENG000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,,CGCON-BSTC01-00007,40.36495,-70.780717,,0,94,
+AR44,,,,CP02PMUI-SBS01-01-MOPAK0000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,,CGINS-MOPAK0-99999,40.36495,-70.780717,,0,94,
+AR44,,,,CP02PMUI-WFP01-00-WFPENG000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGCON-CWFPCP-93601,40.36495,-70.780717,,70,94,
+AR44,,,,CP02PMUI-WFP01-01-VEL3DK000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-VEL3DK-00022,40.36495,-70.780717,,70,94,
+AR44,,,,CP02PMUI-WFP01-02-DOFSTK000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-DOFSTK-02397,40.36495,-70.780717,,70,94,
+AR44,,,,CP02PMUI-WFP01-03-CTDPFK000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-CTDPFK-00106,40.36495,-70.780717,,70,94,
+AR44,,,,CP02PMUI-WFP01-04-FLORTK000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-FLORTK-00969,40.36495,-70.780717,,70,94,
+AR44,,,,CP02PMUI-WFP01-05-PARADK000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-PARADK-20423,40.36495,-70.780717,,70,94,
```
Comment (cdobs) [inline deployment/CP02PMUI_Deploy.csv:136]: Looks good

File: deployment/CP02PMUO_Deploy.csv [hunk 6]
```diff
@@ -116,21 +116,21 @@ AR34,,AR39,,CP02PMUO-WFP01-02-DOFSTK000,13,1,2019-04-23T14:11:00,2019-10-14T12:0
...
+AR44,,,,CP02PMUO-SBS01-00-STCENG000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015,,CGCON-BSTC01-00009,39.942883,-70.770933,,0,443,
+AR44,,,,CP02PMUO-SBS01-01-MOPAK0000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015,,CGINS-MOPAK0-11355,39.942883,-70.770933,,0,443,
+AR44,,,,CP02PMUO-WFP01-00-WFPENG000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015,CGVEH-CWFPCP-10304,CGCON-CWFPCP-10304,39.942883,-70.770933,,419,443,
+AR44,,,,CP02PMUO-WFP01-01-VEL3DK000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015,CGVEH-CWFPCP-10304,CGINS-VEL3DK-00033,39.942883,-70.770933,,419,443,
+AR44,,,,CP02PMUO-WFP01-02-DOFSTK000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015,CGVEH-CWFPCP-10304,CGINS-DOFSTK-02726,39.942883,-70.770933,,419,443,
+AR44,,,,CP02PMUO-WFP01-03-CTDPFK000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015,CGVEH-CWFPCP-10304,CGINS-CTDPFK-00127,39.942883,-70.770933,,419,443,
+AR44,,,,CP02PMUO-WFP01-04-FLORTK000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015,CGVEH-CWFPCP-10304,CGINS-FLORTK-01100,39.942883,-70.770933,,419,443,
+AR44,,,,CP02PMUO-WFP01-05-PARADK000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015,CGVEH-CWFPCP-10304,CGINS-PARADK-20582,39.942883,-70.770933,,419,443,
```
Comment (cdobs) [inline deployment/CP02PMUO_Deploy.csv:136]: Looks good

File: deployment/CP03ISPM_Deploy.csv [hunk 7]
```diff
@@ -1,28 +1,28 @@
...
+AR39,,AR44,,CP03ISPM-SBS01-01-MOPAK0000,3,1,2019-10-06T13:41:00,2020-06-13T19:24:00,CGMCP-03ISPM-00003,,CGINS-MOPAK0-58481,40.364817,-70.888668,,0,92,
+AR39,,AR44,,CP03ISPM-WFP01-00-WFPENG000,3,1,2019-10-06T13:41:00,2020-06-13T19:24:00,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGCON-CWFPCP-30001,40.364817,-70.888668,,68,92,
+AR39,,AR44,,CP03ISPM-WFP01-01-VEL3DK000,3,1,2019-10-06T13:41:00,2020-06-13T19:24:00,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-VEL3DK-00468,40.364817,-70.888668,,68,92,
+AR39,,AR44,,CP03ISPM-WFP01-02-DOFSTK000,3,1,2019-10-06T13:41:00,2020-06-13T19:24:00,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-DOFSTK-03506,40.364817,-70.888668,,68,92,
+AR39,,AR44,,CP03ISPM-WFP01-03-CTDPFK000,3,1,2019-10-06T13:41:00,2020-06-13T19:24:00,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,40.364817,-70.888668,,68,92,
+AR39,,AR44,,CP03ISPM-WFP01-04-FLORTK000,3,1,2019-10-06T13:41:00,2020-06-13T19:24:00,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,40.364817,-70.888668,,68,92,
+AR39,,AR44,,CP03ISPM-WFP01-05-PARADK000,3,1,2019-10-06T13:41:00,2020-06-13T19:24:00,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-PARADK-20541,40.364817,-70.888668,,68,92,
```
Comment (cdobs) [inline deployment/CP03ISPM_Deploy.csv:28]: Looks good

File: deployment/CP04OSPM_Deploy.csv (hunk 1 above)
Comment (cdobs) [inline deployment/CP04OSPM_Deploy.csv:111]: Looks good

File: deployment/CP05MOAS-PG583_Deploy.csv [hunk 8]
```diff
@@ -1,8 +1,15 @@
...
+TS20200613,,,,CP05MOAS-PG583-00-ENG000000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGCON-GLDRCP-00583,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-01-CTDGVM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-CTDGVM-09307,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-02-DOSTAM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-DOSTAM-00539,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-03-FLORTM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-FLORTM-04093,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-04-FLORTO000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-FLORTO-01348,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-05-NUTNRM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-NUTNRM-00709,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-06-PARADM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-PARADM-50218,40.08518,-70.36618,,200,200,CNPM
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-PG583_Deploy.csv:15]: looks good!

**Pull request merged**